<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSideWidget</class>
 <widget class="QBaseWidget" name="QSideWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QSideWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSideWidget{
	background:white;
}
QBaseWidget#subWidget{
	background:#e7e7e7;
}
QTextEdit#textEdit{
	background:transparent;
	border:none;
	font:12px &quot;微软雅黑&quot;;
}
QToolButton#btnEdt{
	border-image: url(:/images/edit);
}
QTabWidget::tab-bar{
	border-width:0px;
	background:white;
	alignment:center;
}
QTabBar::tab{
	border-bottom-color: #C2C7CB;
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    max-width: 75px;
	min-width:75px; 
	height:40px;
    font:30 16px &quot;微软雅黑&quot;;
    padding: 0px;
	beckground:white;
}

QTabBar::tab:selected{
	border-color: white;
	background:white;
	color:green;
}

QTabWidget::pane{
	border:none;
	background:white;
}
QListWidget{
	border:none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QBaseWidget" name="subWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>250</width>
        <height>60</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="placeholderText">
       <string>请编辑简介</string>
      </property>
     </widget>
     <widget class="QToolButton" name="btnEdt">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>35</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="autoRepeatDelay">
       <number>300</number>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabCommit">
      <attribute name="title">
       <string>提交</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabLookup">
      <attribute name="title">
       <string>查询</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabLog">
      <attribute name="title">
       <string>日志</string>
      </attribute>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>201</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QBaseWidget</class>
   <extends>QWidget</extends>
   <header>qbasewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../microlib.qrc"/>
 </resources>
 <connections/>
</ui>
